### PR TITLE
LPS-162661 Add help property to specific fields in the frontend

### PR DESCRIPTION
**Pull request by claraizquierdo**
Motivation
=======
Help icon is missing from WebDAV field
[Link to story or ticket](https://issues.liferay.com/browse/LPS-162661)

Proposed Solution
========
The specific fields of an asset can now have a help attribute that must display a help icon with a Tooltip.

Steps to Verify:
----------------
- Create a basic document
- Go to Content Dashboard (Liferay Menu > Applications > Content > Content Dashboard)
- Click in the info icon of the document in the table
- In the sidebar click on the details accordion component
- Go to WebDAV URL and check there is a help icon that display a tooltip

### CODE REVIEW

File: modules/apps/content-dashboard/content-dashboard-web/types/src/main/resources/META-INF/resources/js/components/SidebarPanelInfoView/SpecificFields.d.ts [hunk 1]
```diff
@@ -19,6 +19,7 @@ declare const SpecificFields: ({
 }: IProps) => (string | JSX.Element)[] | null;
 declare type SpecificItemTypes = 'Date' | 'String' | 'URL';
 interface SpecificField {
+	help: string;
```
[the remark this answers is not in the document]
Comment (claraizquierdo) [reply modules/apps/content-dashboard/content-dashboard-web/types/src/main/resources/META-INF/resources/js/components/SidebarPanelInfoView/SpecificFields.d.ts:22]: Yes! it is not required. I will change it when we integrate the back en the front

**Comment by cgoncas** [comment]
Hey @claraizquierdo 

I have started working on the Backend solution in https://github.com/liferay-tango/liferay-portal/pull/3000.

Thanks!!

**Comment by cgoncas** [comment]
Hey @claraizquierdo !!

Could you use this play https://liferay.atlassian.net/wiki/spaces/ENGTANGO/pages/1368786134/Play+Pull+Request+Reviews and fill the description of this PR.

We should not work in PR that doesn't have a description (@beltranrengifo)

Thanks!!

**Comment by cgoncas** [comment]
ci:test:sf

**Comment by cgoncas** [comment]
ci:test:relevant

**Comment by cgoncas** [comment]
ci:test:sf

**Comment by cgoncas** [comment]
ci:test:relevant

**Comment by cgoncas** [comment]
ci:test:tango

**Comment by claraizquierdo** [comment]
ci:test:relevant

**Comment by claraizquierdo** [comment]
ci:test:sf

**Comment by claraizquierdo** [comment]
ci:stop

**Comment by claraizquierdo** [comment]
ci:test:sf

**Comment by claraizquierdo** [comment]
ci:test:relevant

**Comment by claraizquierdo** [comment]
ci:test:relevant

**Review by Mikellorza — APPROVED** [review]

**Comment by claraizquierdo** [comment]
ci:reevaluate:1307401_5319

**Comment by claraizquierdo** [comment]
ci:test:tango

**Comment by claraizquierdo** [comment]
@cgoncas is the backend part ready to be forwarded?

**Comment by cgoncas** [comment]
ci:forward

**Pull request closed without merging**